backend/Dockerfiles/data/secret-patterns

```diff
@@ -10,7 +10,7 @@ EAACEdEose0cBA[0-9A-Za-z]{16,255}
 [s|S][e|E][c|C][r|R][e|E][t|T](.){1,64}['|"][0-9a-zA-Z]{32,45}['|"]
 [0-9]+-[0-9A-Za-z_]{32}\.apps\.googleusercontent\.com
 ya29\.[0-9A-Za-z\-_]{55}
-(ftp|ftps|http|https|mongodb|mongodb\+srv|postgres):\/\/[^/:@[:space:]]{3,64}:[^/:@[:space:]]{3,64}@[a-zA-Z0-9\.-]+(:[0-9]{1,5}){0,1}
+(ftp|ftps|http|https|mongodb|mongodb\+srv|postgres)://[^/:@[:space:]]{3,64}:[^/:@[:space:]]{3,64}@[a-zA-Z0-9\.-]+(:[0-9]{1,5}){0,1}
```

Comment:
This doesn't change any functionality, but grep warns about these lines otherwise (and we filter those warnings out, so it's just a small optimization and to clean up output)